build.gradle.kts

```diff
@@ -209,7 +209,7 @@ subprojects {
 
   tasks.withType<Detekt> {
     dependsOn(":detektive:assemble")
-    exclude { it.file.absolutePath.contains("/generated/source/") }
+    exclude { it.file.absolutePath.contains("/generated/source/") || it.file.absolutePath.contains("SampledLogger") }
```

Comment:
The `SampledLogger` class makes detekt really unhappy with a recursive something something error. Couldn't figure out why and decided to exclude this file.